kubectl-cloudflow/config/config.go

```diff
@@ -135,22 +136,22 @@ func loadAndMergeConfigs(configFiles []string) (*Config, error) {
 	hoconConf := config.parse()
 
 	// Maybe move validation completely to operator. The CLI can check for status. Bad side effect maybe, is that there will be incorrect resources in K8s.
-	if hoconConf.GetConfig(cloudflowStreamletsPrefix) == nil && hoconConf.GetConfig(cloudflowRuntimesPrefix) == nil {
-		return nil, fmt.Errorf("configuration does not contain '%s' or '%s' config sections", cloudflowStreamletsPrefix, cloudflowRuntimesPrefix)
+	if hoconConf.GetConfig(cloudflowStreamletsPath) == nil && hoconConf.GetConfig(cloudflowRuntimesPath) == nil {
+		return nil, fmt.Errorf("configuration does not contain '%s' or '%s' config sections", cloudflowStreamletsPath, cloudflowRuntimesPath)
 	}
-	streamletsConfig := hoconConf.GetConfig(cloudflowStreamletsPrefix)
+	streamletsConfig := hoconConf.GetConfig(cloudflowStreamletsPath)
 	if streamletsConfig != nil && streamletsConfig.Root().IsObject() {
```

Comment:
A "TODO: Refactor for readability" could go here. There is a couple of `if`'s that can be stacked I think and a function could be extracted for the inner `for`.